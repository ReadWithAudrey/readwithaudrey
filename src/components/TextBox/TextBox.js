import React from 'react'
import { StyledTextBox } from './TextBox.style'
import PropTypes from 'prop-types'

const TextBox = props => <StyledTextBox>{props.children}</StyledTextBox>

TextBox.propTypes = {
  children: PropTypes.oneOfType([PropTypes.string, PropTypes.array]),
}

export default TextBox
