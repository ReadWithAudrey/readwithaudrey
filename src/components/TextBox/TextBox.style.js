import styled from 'styled-components'

export const StyledTextBox = styled.textarea`
  background: #fff;
  border: #d3d3d3 1px solid;
  padding: 7px;
  margin: 10px 0px;
  border-radius: 10px;
  width: 100%;
`
