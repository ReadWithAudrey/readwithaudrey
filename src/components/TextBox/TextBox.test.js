import React from 'react'
import renderer from 'react-test-renderer'
import TextBox from './TextBox'

describe('TextBox component snapshot test', () =>
  it('renders correctly', () => {
    const tree = renderer.create(<TextBox />).toJSON()
    expect(tree).toMatchSnapshot()
  }))
