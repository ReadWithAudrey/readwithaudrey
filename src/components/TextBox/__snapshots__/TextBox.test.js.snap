// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextBox component snapshot test renders correctly 1`] = `
<p
  className="sc-bdVaJa montserrat f5 tc lh-copy kktwlW"
/>
`;
